apps/labs/posts/sparse-array-ecosystem.md

```diff
@@ -0,0 +1,118 @@
+
+#### `cupyx.sparse` and `scipy.sparse`
+
+One can do any of the following to create a `scipy.sparse` or `cupyx.sparse` array:
+
+1. Use `array_type(existing_obj)`, as [an example CSR](https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.csr_array.html).
+2. (Only for SciPy) Instantiate a [`scipy.sparse.dok_array`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.dok_array.html), populate it, and then use [`scipy.sparse.dok_array.asformat`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.dok_array.asformat.html#scipy.sparse.dok_array.asformat)
+   - For `cupyx.sparse`, we can later move the array to the GPU.
+3. Use `array_type(tuple_of_constituent_arrays)`.
+
+The main pain points for this set of libraries are twofold:
+
+1. They only support 2-D arrays or matrices.
+2. Most of the existing work only exists for the old, deprecated `np.matrix` interface. SciPy is trying to move away from that, but progress there is slow.
+
+#### `torch.sparse`
+
+To create a sparse array, one can:
+
+1. Call the [`torch.Tensor.to_sparse`](https://pytorch.org/docs/stable/generated/torch.Tensor.to_sparse.html) on an existing `Tensor` (or any of the `to_sparse_*` format-specific methods).
+2. By calling the format-specific constructors, for example [`torch.sparse_coo_tensor`](https://pytorch.org/docs/stable/generated/torch.sparse_coo_tensor.html), to create a format from its constituent arrays.
+
+The main pain points for `torch.sparse` is the differing API from most other libraries, and lack of DOK support to construct hyper-sparse tensors intuitively.
+
+### Honorable Mentions
+
+There are a number of honorable mentions that haven't managed to gain enough traction to be included in this list:
+
+- [**`jax.experimental.sparse`**](https://jax.readthedocs.io/en/latest/jax.experimental.sparse.html): A forward-thinking sparse array module based on the [MLIR sparse tensor dialect](https://mlir.llvm.org/docs/Dialects/SparseTensorOps/).
+- [**`pysparse`**](https://pysparse.sourceforge.net/): Not to be confused with `sparse`, this was a package that didn't take off, but has a rather common name.
+- [**TACO**](http://tensor-compiler.org/): A C++ package with Python bindings by some of the brains behind performant sparse code generation.
+- [**`finch-tensor` and `Finch.jl`**](https://willowahrens.io/Finch.jl/dev): A Julia package with respective Python bindings that's also showing promise and contains some novel research on performant sparse computing.
+
+## Conclusion
+
+The Sparse array ecosystem in Python is, in a word, _fragmented_. The meme below illustrates the options available to users. We would like to change that for the better. To that end, we at Quansight along with Professor Samaringhe's group at MIT CSAIL have received a grant from DARPA spanning two years of work to help move the ecosystem in a direction that would benefit the whole commpunity.
```

Comment:
Not clear to me how the meme supports the concept of fragmentation.

Reply:
Removed the meme in https://github.com/Quansight/Quansight-website/pull/837/commits/56e6d1adcbe45f4b1b10464706b750681797fe41.